applications/asset_tracker_v2/src/modules/Kconfig.modem_module

```diff
@@ -28,12 +28,30 @@ config MODEM_AUTO_REQUEST_POWER_SAVING_FEATURES
 	bool "Auto request power saving features"
 	default y
 
-config MODEM_CONVERT_RSRP_AND_RSPQ_TO_DB
```

Comment:
Nothing wrong as such but why do we constantly need to rename symbols ?
This makes it unnecessarily hard for our users to update to later release.

I believe it would be better to deprecate `MODEM_CONVERT_RSRP_AND_RSPQ_TO_DB` first, and until removed completely then if users are selecting this symbol then it will do a:
```
select MODEM_DYNAMIC_DATA_CONVERT_RSRP_TO_DBM
select MODEM_NEIGHBOR_CELLS_CONVERT_RSRQ_TO_DB
```

Reply:
Valid point. Readded the `MODEM_CONVERT_RSRP_AND_RSRQ_TO_DB` option and marked it as "deprecated".